src/util-coredump-config.c

```diff
@@ -116,14 +122,15 @@ int32_t CoredumpLoadConfig (void)
             SCLogInfo ("Unexpected type for rlim_t");
             return 0;
         }
-        errno = 0;
         if (rlim_size == 8) {
-            max_dump = (rlim_t) strtoull (dump_size_config, NULL, 10);
+            ret = ByteExtractStringUint64(
+                    (uint64_t *)&max_dump, 10, strlen(dump_size_config), dump_size_config);
         }
         else if (rlim_size == 4) {
-            max_dump = (rlim_t) strtoul (dump_size_config, NULL, 10);
+            ret = ByteExtractStringUint32(
+                    (uint32_t *)&max_dump, 10, strlen(dump_size_config), dump_size_config);
         }
-        if ((errno == ERANGE) || (errno != 0 && max_dump == 0)) {
+        if ((ret <= 0) || (max_dump == 0)) {
             SCLogInfo ("Illegal core dump size: %s.", dump_size_config);
```

Comment:
0 is valid, for disable core-dump.  

This should probably be a warning, but I realize this was previous behavior.

Reply:
Reference: https://github.com/OISF/suricata/blob/master/suricata.yaml.in#L1351